lib/user.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var UserStore = require('fh-wfm-user/lib/user/store');
```

Comment:
Not sure why the UserStore is coming from the fh-wfm-user module.

Should not be coming from the fh-wfm-store module?